website/templates/util/render_node.mako

```diff
@@ -51,16 +51,26 @@
             % endif
             </span>
 
-            <!-- Show/Hide recent activity log -->
-            % if not summary['archiving']:
```

Comment:
I'm not sure this line should go away. Can you look into what this is doing more? I'm fairly certain certain actions are disabled if a node is mid archive job (when a node is registered).